api/src/main/java/me/lokka30/treasury/api/economy/response/EconomyFailureReason.java

```diff
@@ -40,10 +48,11 @@
          */
         @Override
         public @NotNull String getDescription() {
-            return "That feature is currently not available during migration!";
+            return "That feature is currently not available during migration.";
         }
     },
 
+    // TODO document further. check what this constant would be used for.
```

Comment:
Unresolved TODO here, I assume we do actually want to document this immediately. Request cancellation could be anything that results in a request being cancelled, server shutdown, thread interruption, etc.